index.html

```diff
@@ -74,44 +86,239 @@
+                    </div>
+
+                    <ul class="footer__menu-list footer__menu-list--hidden">
+                        <li class="footer__link">Find a Gift List</li>
+                        <li class="footer__link">Guest T&Cs</li>
+                    </ul>
+
+                </div>
+
+            </div>
+
+            <div class="divider divider--hidden"></div>
+
+            <div class="footer__menu">
+                
+                <div class="footer__menu-section">
+
+                    <div class="heading-container">
+                        <h4 class="footer__menu-heading">Contact Us</h4>
+                    </div>
+
+                    <ul class="footer__menu-list menu-details">
+                        <li class="footer__link-off">020 7384 8400</li>
+                        <li class="footer__link-off">hello@weddingshop.com</li>
+                    </ul>
+
+                </div>
+                
+                <div class="footer__menu-section">
+
+                    <h4 class="footer__menu-heading">Follow Us</h4>
+
+                    <ul class="footer__menu-list">
+                        <a href="https://www.pinterest.co.uk/weddingshopuk/"><img class="footer__socials" src="img/pin.svg"/></a>
+                        <a href="https://facebook.com/theweddingshop"><img class="footer__socials" src="img/fb.svg"/></a>
+                        <a href="https://www.youtube.com/channel/UCID13-6OQt9ubZq2OZR2cvQ/featured"><img class="footer__socials" src="img/yt.svg"/></a>
+                        <a href="https://www.instagram.com/theweddingshopuk/"><img class="footer__socials" src="img/ig.svg"/></a>
+                        <a href="https://twitter.com/"><img class="footer__socials" src="img/tt.svg"/></a>
+                    </ul>
+
+                </div>
+
+                <div class="footer__menu-section footer__menu-section--narrow">
+
+                    <h4 class="footer__menu-heading">Request A Brochure</h4>
+                    <ul class="footer__menu-list">
+                        <li class="footer__link-button">Send Me One</li>
+                    </ul>
+
+                </div>
+                
+                <div class="footer__menu-section footer__menu-section--narrow">
+
+                    <h4 class="footer__menu-heading">Download Our App</h4>
+                    <ul class="footer__menu-list">
+                        <li><img class="footer__link-app" src="img/App Store.svg"/></li>
+                    </ul>
+
+                </div>
+               
+            </div>
+
+            <div class="divider"></div>
+
+            <div class="footer__bottom">
+
+                <a class="footer__terms">Terms & Conditions</a>
+                <a class="footer__terms">Privacy & Cookie Policy</a>
+                <a class="footer__terms">Contact</a>
+                <a class="footer__terms footer__terms-copyright">© The Wedding Shop <span id="copyrightYear"></span></a>

```

Comment:
it doesn't really matter but instead of searching a character that might not be present in your keyboard and copying it from somewhere, you might want to use something called HTML entity. In this case you could use `&copy;`.